example/components/ToastCode.tsx

```diff
@@ -1,21 +1,11 @@
 import * as React from 'react';
 
-import { themes } from './constants';
+import { themes, typeMap } from './constants';
+
+
 
 function getType(type: string) {
-  switch (type) {
-    case 'default':
-    default:
-      return 'toast';
-    case 'success':
-      return 'toast.success';
-    case 'error':
-      return 'toast.error';
-    case 'info':
-      return 'toast.info';
-    case 'warning':
-      return 'toast.warn';
-  }
+  return typeMap[type] || 'toast';
```

Comment:
using the OR operator to default to string literal 'toast', but typeMap already has a default field set to 'toast'.

`return typeMap[type] || typeMap.default`

Reply:
PR feedback incorporated could you please check again?